packages/chakra-ui/src/FieldTemplate/FieldTemplate.tsx

```diff
@@ -25,13 +23,19 @@ const FieldTemplate = (props: FieldTemplateProps) => {
     schema,
     uiSchema,
   } = props;
+  const uiOptions = getUiOptions(uiSchema);
+  const WrapIfAdditionalTemplate = getTemplate(
```

Comment:
Adding the string name as the generic will ensure the proper typing
```suggestion
  const WrapIfAdditionalTemplate = getTemplate<"WrapIfAdditionalTemplate">(
```